File: server/application/src/main/java/de/codecentric/iot/rapiro/blazeds/destinations/VisionPublishingActor.java
package de.codecentric.iot.rapiro.blazeds.destinations;

import de.codecentric.iot.rapiro.blazeds.actors.BlazeDsPublishingActor;
import org.springframework.context.annotation.Scope;
import org.springframework.stereotype.Component;

/**
 * Created by christoferdutz on 18.10.16.
 */
@Scope("prototype")
@Component("visionPublishingActor")
public class VisionPublishingActor extends BlazeDsPublishingActor {

    public VisionPublishingActor() {
        super("visionEvents");
    }

}

File: server/application/src/main/scala/de/codecentric/iot/rapiro/blazeds/destinations/VisionPublishingActor.scala
package de.codecentric.iot.rapiro.blazeds.destinations

import de.codecentric.iot.rapiro.blazeds.actors.BlazeDsPublishingActor
import org.springframework.context.annotation.Scope
import org.springframework.stereotype.Component

/**
  * Created by christoferdutz on 24.10.16.
  */
@Scope ("prototype")
@Component ("visionPublishingActor")
class VisionPublishingActor extends BlazeDsPublishingActor {

  override def destinationName: String = "visionEvents"

}
